packages/chat/src/components/NewChat/INewChat.ts

```diff
@@ -0,0 +1,32 @@
+import type { Nullish } from '@/types';
+
+import type { HeaderProps } from '../Header';
+import type { INewFooter } from '../NewFooter';
+import type { IWelcomeMessage } from '../WelcomeMessage';
+
+export interface INewChat extends HeaderProps, IWelcomeMessage, INewFooter, React.PropsWithChildren<unknown> {
```

Comment:
Copied the props from the 'old' chat props - we'll start with this, and when it fully works we can refactor :)